commit f4cc905c69ca69c68cb95674cefce4abc37ce046
Author: wangqr <wangqr@wangqr.tk>
Date:   Mon Feb 17 14:42:07 2020 +0800

    Use target name without directory in $*_OBJ macro
    
    Fix Aegisub/Aegisub#171

diff --git a/Makefile.target b/Makefile.target
index 516ef3c24..5c4c5d259 100644
--- a/Makefile.target
+++ b/Makefile.target
@@ -112,7 +112,7 @@ POST_FLAGS = $($@_FLAGS) -c -o $@ $<
 # Libraries contain all object files they depend on (but they may depend on other files)
 # Not using libtool on OS X because it has an unsilenceable warning about a
 # compatibility issue with BSD 4.3 (wtf)
-lib%.a: $$($$*_OBJ)
+lib%.a: $$($$(*F)_OBJ)
 	@$(BIN_MKDIR_P) $(dir $@)
 	$(BIN_AR) cru $@ $(filter %.o,$^)
 	$(BIN_RANLIB) $@
